### DISCUSSION TITLE
Can we use the old simple gallois theme?

### DISCUSSION THREAD

**Discussion by ziyuang** [ohmyzsh/ohmyzsh#11886] [Help] [answered]
I chose gallois for its simplicity, but now it [includes](https://github.com/ohmyzsh/ohmyzsh/commit/76e541a3fe71075c8ef75ecf44cd60f17b5292bb) extra information that I don't need and occupies more space. Do you know if I can use the old theme or is there any equivalent? Thank you.

**Comment by carlosala** [comment, marked as answer]
You can use `oldgallois` theme.
